Formula/dar.rb

```diff
@@ -27,11 +40,18 @@ class Dar < Formula
   depends_on "upx" => [:build, :optional]
 
   def install
+    if build.stable? && ENV.compiler == :clang && MacOS.version >= :mavericks
+      ENV.libstdcxx
+    elsif build.devel?
+      ENV.O0
+      ENV.deparallelize
```

Comment:
Weird. Mind checking gcc and gcc5 (I need O0 for both). I'm wondering if it's because the machine is Skylake, but that didn't seem likely since setting -march core2 instead of -march native for optimization didn't help, but that doesn't rule it out definitively.